Formula/qt5.rb

```diff
@@ -86,6 +87,7 @@ class Qt5 < Formula
   depends_on :mysql => :optional
   depends_on :postgresql => :optional
   depends_on :xcode => :build
+  depends_on "openssl" => :optional
```

Comment:
And I assume getting mysql or postgresql to use SecureTransport is a fool's errand?